.github/ISSUE_TEMPLATE/bug_report.yaml

```
@@ -0,0 +1,49 @@
+name: Bug Report
+description: Create a report to help us improve
+body:
+  - type: markdown
+    attributes:
+      value: |
+        Thanks for taking the time to fill out this bug report!
+
+  - type: textarea #description
+    attributes:
+      label: Describe the bug
+      description: A clear and concise description of what the bug is.
+      placeholder: Tell us what you see!
+    validations:
+      required: true
+
+  - type: textarea #reproduce
+    attributes:
+      label: Environment
+      description:
+        "The OS + version you’re running Kubescape on, e.g Ubuntu 22.04 LTS
+        Version: the version that Kubescape reports when you run `kubescape version`"
+      placeholder: Kindly elaborate.
+
+  - type: textarea #description
+    attributes:
+      label: Steps To Reproduce
+      description: "Steps to reproduce the behavior:
+1. Go to '...'
+2. Click on '....'
+3. Scroll down to '....'
+4. See error"
```

issue: Github flattens this list into a single line when rendering the description

It would be better to have it as a placeholder for the textarea.

```suggestion
      description: "Steps to reproduce the behavior"
      placeholder: |
        1. Go to '...'
        2. Click on '....'
        3. Scroll down to '....'
        4. See error"
```